libs/wingc/src/visit.rs

```diff
@@ -206,7 +206,7 @@ where
 {
 	v.visit_symbol(&node.name);
 
-	v.visit_constructor(&node.constructor);
+	v.visit_constructor(&node.initializer);
```

Comment:
We also need to visit the inflight initializer?

(I see this is something we should automate, I've created an issue for it for now https://github.com/winglang/wing/issues/2043)